### ISSUE TITLE
Failed to resolve the dependency- 1.2.5

### ISSUE THREAD

**Issue by symtalha14** [harjot-oberai/MaterialShadows#23]
Hi,

Please confirm when this will be pushed to mavenCentral, since jCenter is no longer working.

Thanks

`Failed to resolve: com.sdsmdg.harjot:materialshadows:1.2.5
`

**Comment by harutlc** [comment]
Hi @symtalha14 
could you resolve this issue?

**Comment by symtalha14** [comment]
No, it seems to be completely out of maintenance. We removed it from our project.
